llvm/lib/Target/AIE/AIEBaseRegisterInfo.h

```diff
@@ -47,6 +47,11 @@ struct AIEBaseRegisterInfo : public TargetRegisterInfo {
   bool isSimplifiableReservedReg(MCRegister PhysReg) const override {
     return false;
   }
+
+  // Whether a reserved register is sticky
```

Comment:
Perhaps add a definition of 'sticky' here.  In particular, I think that the stickiness is a priori a property of an operand in an instruction. A reset of a flag is a non-sticky operation.